extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/linkHandler/SoundcloudChannelLinkHandlerFactory.java

```diff
@@ -21,6 +21,8 @@ public static SoundcloudChannelLinkHandlerFactory getInstance() {
     @Override
     public String getId(String url) throws ParsingException {
         Utils.checkUrl(URL_PATTERN, url);
+        // Remove the tailing slash from URLs due to issues with the SoundCloud API
+        if (url.charAt(url.length() -1) == '/') url = url.substring(0, url.length()-1);
```

Comment:
Why not moving this inside `resolveIdWithEmbedPlayer`?

Reply:
I think I was just mimicking what I did in 6a70cb9d50132c1cf1bf59478f029b199d65fefc
but I think I can move everything to that instead

I don't really remember my branches that well anymore, but I think I got it